internal/store/nats.go

```diff
@@ -22,8 +22,8 @@ import (
 )
 
 var (
-	bucketName = "active-conditions"
-	bucketOpts = []kv.Option{
+	ActiveConditionBucket = "active-conditions"
```

Comment:
why is this bucket being exported from the module?

Reply:
I was working on some tests that are now pushed https://github.com/metal-toolbox/conditionorc/blob/961699928799ae5d6a1d43de442ed848b008fcc3/internal/orchestrator/updates_test.go#L147